apps/perms/serializers/user_permission.py

```diff
@@ -51,7 +51,7 @@ class NodePermedSerializer(serializers.ModelSerializer):
     class Meta:
         model = Node
         fields = [
-            'id', 'name', 'key', 'value',
+            'id', 'name', 'key', 'value', 'full_value',
             'org_id', "assets_amount"
         ]
         read_only_fields = fields
```

Comment:
I cannot review your provided code without the actual snippet of it being presented. However, typically when comparing two classes (like you did), you would look at their field definitions and possibly any related methods which could help with identifying differences or optimizations based on those details. Always note that to effectively diagnose problems or offer advice, one needs actual context about what is happening within the class(es).